doc/en/components/grids/pivot-grid/overview.md

```diff
@@ -2,13 +2,368 @@
+            member: 'AmountOfSale',
+            displayName: 'Amount of Sale',
+            aggregate: {
+                key: 'SUM',
+                aggregator: IgxTotalSaleAggregate.totalSale,
+                label: 'Sum of Sale'
+            },
+            aggregateList: [{
+                key: 'SUM',
+                aggregator: IgxTotalSaleAggregate.totalSale,
+                label: 'Sum of Sale'
+            }, {
+                key: 'MIN',
+                aggregator: IgxTotalSaleAggregate.totalMin,
+                label: 'Minimum of Sale'
+            }, {
+                key: 'MAX',
+                aggregator: IgxTotalSaleAggregate.totalMax,
+                label: 'Maximum of Sale'
+            }]
+        }
+    ]
+}
+
+public static totalSale: PivotAggregation = (members, data: any) =>
+    data.reduce((accumulator, value) => accumulator + value.UnitPrice * value.UnitsSold, 0);
+
+public static totalMin: PivotAggregation = (members, data: any) => {
+    return data.map(x => x.UnitPrice * x.UnitsSold).reduce((a, b) => Math.min(a, b));
+};
+
+public static totalMax: PivotAggregation = (members, data: any) => {
+    return data.map(x => x.UnitPrice * x.UnitsSold).reduce((a, b) => Math.max(a,b));
+};
+```
+
+```razor
+@code {
+    IgbPivotConfiguration pivotConfiguration1 = new IgbPivotConfiguration();
+    IgbPivotValue pivotValue = new IgbPivotValue()
+        {
+            Member = "Sales",
+            Name = "pivotValue1",
+            DisplayName = "Amount of Sales",
+            Enabled = true,
+            Aggregate = new IgbPivotAggregator() { Key = "sum", AggregatorName = PivotAggregationType.SUM, Label = "Sum of Sales" }
+        };
+    pivotValue.AggregateList.Add(new IgbPivotAggregator() { Key = "sum", AggregatorName = PivotAggregationType.SUM, Label = "Sum of Sales" });
+    pivotValue.AggregateList.Add(new IgbPivotAggregator() { Key = "min", AggregatorName = PivotAggregationType.MIN, Label = "Minimum of Sales" });
+    pivotValue.AggregateList.Add(new IgbPivotAggregator() { Key = "max", AggregatorName = PivotAggregationType.MAX, Label = "Maximum of Sales" });
+    pivotConfiguration1.Values.Add(pivotValue);
+```
+
+The pivot value also provides a `DisplayName` property. It can be used to display a custom name for this value in the column header.
+
+### Enable property
+
+`IPivotConfiguration` is the interface that describes the current state of the `PivotGrid` component. With it the developer can declare fields of the data as **rows**, **columns**, **filters** or **values**. The configuration allows enabling or disabling each of these elements separately. Only enabled elements are included in the current state of the pivot grid. The `PivotDataSelector` component utilizes the same configuration and shows a list of all elements - enabled and disabled. For each of them there is a checkbox in the appropriate state. End-users can easily tweak the pivot state by toggling the different elements using these checkboxes.
```

Comment:
For Blazor i think the names of the interfaces should be replaced with the `IgbPivotConfiguration`, `IgbPivotDimension` and `IgbPivotValue` classes